site/content/docs/11-deploying.md

```diff
@@ -2,7 +2,21 @@
 title: Deployment
 ---
 
-Sapper apps run anywhere that supports Node 8 or higher.
+Sapper apps run anywhere that supports Node 10 or higher.
+
+### Deploying from sapper build
+
+We will need to have `server.js` dependencies available on the production machine. You can either install the dependencies locally or have the `package.json` and `package-lock.json` available on the production machine and install the dependencies there. Production dependencies can be installed with:
```

Comment:
The text switches back and forth between "we" and "you", which reads funny. Let's stick with one or the other for consistency. It seems like the rest of the docs you "you" more than "we"

```suggestion
You will need to have `server.js` dependencies available on the production machine. You can either install the dependencies locally or have the `package.json` and `package-lock.json` available on the production machine and install the dependencies there. Production dependencies can be installed with:
```